apps/docs/content/3-components/2-library/tooltip.mdx

```diff
@@ -7,7 +7,8 @@ libraries:
     link: '?path=/docs/application-tooltip--docs'
     status: 'stable'
   - platform: 'global'
-    status: 'considering'
+    status: 'beta'
+    link: '?path=/docs/application-tooltip--docs'
   - platform: 'react'
     status: 'beta'
     link: ?path=/docs/application-tooltip--docs'
```

Comment:
missing `'`

Reply:
Considering the definition below of `Components/Tooltip` I think that the link should be `?path=/docs/components-tooltip--docs`